src/components/Controls.tsx

```diff
@@ -164,6 +164,7 @@ export default function Controls({ state, setState, mapbox }: ControlsProps) {
               state.input.mode === 'location' ? search : state.input.search
             }
           />
+          <input type="submit" hidden />
```

Comment:
Makes sense to add this.
Also not sure how this was working before the enter key press behavior seems sort of unpredictable currently.

Reply:
I think maybe because there are buttons in the <form> field the browser might be interpreting one of those as a submit?